packages/babel-helper-hoist-variables/src/index.ts

```diff
@@ -38,8 +38,16 @@ const visitor = {
     > = path.get("declarations");
```

Comment:
The hoistVariable visitor should be merged with `environmentVisitor`, too.

Reply:
Lets do it in another PR since it's separate from this bug :+1: